stdlib/public/runtime/AccessibleFunction.cpp

```diff
@@ -130,6 +119,27 @@ void swift::addImageAccessibleFunctionsBlockCallback(
   addImageAccessibleFunctionsBlockCallbackUnsafe(baseAddress, functions, size);
 }
 
+// TODO(distributed): expose dumping records via a flag
+static void
+_dumpAccessibleFunctionRecords() {
+  auto &S = Functions.get();
+
+  fprintf(stderr, "==== Accessible Function Records ====\n");
+  int count = 0;
+  for (const auto &section : S.SectionsToScan.snapshot()) {
+    for (auto &record : section) {
+      auto recordName =
+          swift::Demangle::makeSymbolicMangledNameStringRef(record.Name.get());
+      fprintf(stderr, "Record name: %s\n", recordName.data());
+      fprintf(stderr, "    Function Ptr: %p\n", record.Function.get());
+      fprintf(stderr, "    Flags.IsDistributed: %d\n", record.Flags.isDistributed());
+      ++count;
+    }
+  }
+  fprintf(stderr, "Record count: %d\n", count);
+  fprintf(stderr, "==== End of Accessible Function Records ====\n");
```

Comment:
If you want to keep this around please use `llvm::errs()` instead.

Reply:
Yeah comes in handy, will do

Reply:
This is runtime, seems we're using `  fprintf(stderr` consistently here and not llvm:errs?

Reply:
Hm, alright then